local/test-binary

```diff
@@ -16,10 +16,24 @@ set -o errexit
 set -o xtrace
 
 WORKDIR_ABS=$(cd ${1:-./build}; pwd -P)
+SOURCEDIR=$2
 rm -fr ${WORKDIR_ABS}/PS
 mkdir -p ${WORKDIR_ABS}/PS/sql
 tar -C ${WORKDIR_ABS}/PS --strip-components=1 -zxpf $(ls $WORKDIR_ABS/*.tar.gz | head -1)
 
+if [[ "${SOURCEDIR}" != "" ]]; then
```

Comment:
``if [[ -n ${SOURCEDIR} ]]``

Reply:
Why do you prefer `-n`?
I prefer `[[ STRING != STRING ]]` because of readability.